mariadb/values.yaml

```diff
@@ -7,6 +7,9 @@ name: DEFINED-IN-COMPONENT-CHART
 image: mariadb:10.1
 port_public: 3306
 #
-#root_password: 
+#root_password:
 #initdb_configmap:
 #persistance_claim:
+backup:
+  enabled: true
+  image: hub.global.cloud.sap/c5211168/monsoon3-backup:0.1.2
```

Comment:
This needs to be configurable with a value. The default should be a public image sapcc/backup-tools:v0.1.2. Internally we overwrite this with out image from hub.global.cloud.sap.